inc/utils/namespace.php

```diff
@@ -223,3 +242,53 @@ function merge_aggregates( array $current, array $new, string $bucket_type = ''
 
 	return $merged;
 }
+
+/**
+ * Check type of field.
+ *
+ * @param string $field The full field name.
+ * @param string $type One of 'string', 'number' or 'date'.
+ * @return bool True if type matches field name.
+ */
+function field_type_is( string $field, string $type ) : bool {
```

Comment:
Not sure I see the point of this function?

Reply:
Yeah I was working with a different approach at first, guess it just looked neater but it is essentially superfluous. Can switch to just using the below function.